jasmine.json

```diff
@@ -4,7 +4,6 @@
         "**/*.spec.js"
     ],
     "helpers": [
```

Comment:
Could this whole `helper` entry be deleted?

Reply:
yeah, thought about leaving it just in case `helpers` was ever needed, but that seems unlikely, removed.